testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
python should take care of that in the garbage collection for us. The teardown doesn't make much sense for us. There are rare cases where the teardown would make sense in a python world anyway.